### PR TITLE
completed adding profile to database

### PR DESCRIPTION
**Pull request by ciannadogan**

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+    allow do
+      origins '*'  # <- change this to allow requests from any domain while in development.
+  
+      resource '*',
+        headers: :any,
+        methods: [:get, :post, :put, :patch, :delete, :options, :head]
+    end
+  end
+
```
Comment (WalkerTexas-coder) [inline config/initializers/cors.rb:15]: Are you sure you need cors?

**Review by WalkerTexas-coder — COMMENTED** [review]
Question

File: config/initializers/cors.rb (hunk 1 above)
Comment (brandonho2000) [reply config/initializers/cors.rb:15]: yes. it was erroring out when we ran it, so we added it in to fix the database errors. we added it with out mentors.

Comment (archae0pteryx) [reply config/initializers/cors.rb:15]: We were having some CORs issues and followed some instructions they had found for how to skip the cors check. This bit of code was included in that description we followed.

File: app/javascript/components/api.js [hunk 2]
```diff
@@ -16,3 +16,23 @@ export async function fetchProfile (id) {
...
+  try {
+    const options = {
+      method:'POST',
+      headers:{
+        'Content-Type':'application/json'
+      },
+      body:JSON.stringify(profileData)
+    }
+    const response = await fetch(`profiles`, options)
+      // return response.json()
```
Comment (archae0pteryx) [inline app/javascript/components/api.js:30]: This response is in fact the new profile... we prolly dont need it here so i think we can just return true or something and remove the const assignment... so just put the `await fetch...` and under `return true`

**Review by archae0pteryx — APPROVED** [review]

File: config/initializers/cors.rb (hunk 1 above)
Comment (WalkerTexas-coder) [reply config/initializers/cors.rb:15]: cors should be for separate frontend and backend applications. As a monolithic application cors shouldn't be doing anything for you. 

The skip_before_action :verify authenticity token should raise issues if you don't have it. Can you try to run the app with that command installed w/o cors and let me know what happens?

Comment (WalkerTexas-coder) [reply config/initializers/cors.rb:15]: hmmm okay

Comment (WalkerTexas-coder) [reply config/initializers/cors.rb:15]: I'd like sarah to get eyes on this. We'll be more available in a bit.

**Pull request merged**